### PR TITLE
Fixed task-api if creator is null

### PR DESCRIPTION
**Pull request by wachterjohannes**
This PR fixes the API if a task without a creator will be serialized

### CODE REVIEW

File: Entity/Task.php [hunk 1]
```diff
@@ -215,14 +215,22 @@ public function setTaskId($taskId)
      */
     public function getCreatorFullName()
     {
-        return $this->getCreator()->getFullName();
+        if (!($creator = $this->getCreator())) {
```
Comment (danrot) [inline Entity/Task.php]: Why not simply `!$this->creator`? Or if you don't want to use the protected member variable of the trait something like this:
```php
$creator = $this->getCreator();
if (!$creator) {
    return '';
}

return $creator->getFullName()
```

Negating and assigning in a single if statement does not look really nice to me...

File: Entity/Task.php [hunk 2]
```diff
@@ -215,14 +215,22 @@ public function setTaskId($taskId)
...
+        return $creator->getFullName();
     }
 
     /**
      * {@inheritdoc}
      */
     public function getChangerFullName()
     {
-        return $this->getChanger()->getFullName();
+        if (!($changer = $this->getChanger())) {
```
Comment (danrot) [inline Entity/Task.php]: Same as above

**Comment by wachterjohannes** [comment]
@danrot fixed

**Comment by danrot** [comment]
@wachterjohannes Use your own fork the next time 😛

**Pull request merged**